src/main/java/org/broadinstitute/consent/http/models/tdr/ApprovedUser.java

```diff
@@ -0,0 +1,20 @@
+package org.broadinstitute.consent.http.models.tdr;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ApprovedUser {
+    @JsonProperty
```

Comment:
I'm 99% sure we don't need to use `@JsonProperty` if we serialize these objects in the response using `Gson`